### PR TITLE
update python readme

### PR DESCRIPTION
**Pull request by d6t5**
making python readme consistent with docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,11 +42,13 @@ evervault.api_key = <YOUR-API-KEY>
 result = evervault.encrypt_and_run(<CAGE-NAME>, { 'hello': 'World!' })
 ```
 
-## API Reference
+## Reference
+
+At present, there are two functions available in the Python SDK: `evervault.encrypt()`, `evervault.run()`, and `evervault.encryptAndRun()`.
```
Comment (jonnyom) [inline README.md:47]:
```suggestion
```

This proves my point exactly, it has more than two methods, which we missed, and additionally also has `cage_list`

File: README.md [hunk 2]
```diff
@@ -35,11 +42,13 @@ evervault.api_key = <YOUR-API-KEY>
...
 
-### evervault.encrypt
+### evervault.encrypt()
 
-Encrypt lets you encrypt data for use in any of your evervault cages. You can use it to store encrypted data to be used in a cage at another time.
+`evervault.encrypt()` encrypts data for use in your [Cages](https://docs.evervault.com/tutorial). To encrypt data at the server, simply pass a `dict` or `string` into the `evervault.encrypt()` function. Store the encrypted data in your database as normal.
```
Comment (jonnyom) [inline README.md:51]:
```suggestion
`evervault.encrypt()` encrypts data for use in your [Cages](https://docs.evervault.com/tutorial). To encrypt data at the server, simply pass a `dict` or `str` into the `evervault.encrypt()` function. Store the encrypted data in your database as normal.
```

File: README.md (hunk 1 above)
Comment (d6t5) [reply README.md:43]: what should be added to the `cage_list` section?

Comment (d6t5) [reply README.md:43]: added the section but marked as TODO

Comment (jonnyom) [reply README.md]: It's more or less the same functionality as the ruby one.

```
Evervault.cage_list
Return a CageList object, containing a list of your team's Cages
```

File: README.md [hunk 3]
```diff
@@ -83,3 +88,15 @@ evervault.encrypt_and_run(cageName = str, data = dict[, options = dict])
 | cageName | `str` | Name of the Cage to be run |
 | data | `dict` | Data to be encrypted |
 | options | `dict` | [Options for the Cage run.](#Cage-Run-Options) |
+
+### evervault.cage_list()
```
Comment (jonnyom) [inline README.md:92]:
```suggestion
### evervault.cages()
Returns `CageList` containing all your team's cages
```

Something like that

**Review by edmundojr — APPROVED** [review]

**Review by jonnyom — CHANGES_REQUESTED** [review]
cage_list needs to be updated. It's cages

**Review by jonnyom — APPROVED** [review]

**Pull request merged**